rundeck-data-model/src/main/java/org/rundeck/app/data/model/v1/execution/RdReferencedExecution.java

```diff
@@ -0,0 +1,13 @@
+package org.rundeck.app.data.model.v1.execution;
+
+import org.rundeck.app.data.model.v1.job.JobData;
+
+import java.util.List;
+
+public interface RdReferencedExecution {
+
+    JobData getScheduledExecution();
+    String getStatus();
+    Execution getExecution();
```

Comment:
Should be an id to the execution for the same reason as stated for JobData.

Reply:
Done